query03.sql

```diff
@@ -0,0 +1,21 @@
+/*
+Using the Philadelphia Water Department Stormwater Billing Parcels dataset, pair each parcel with its closest bus stop.
+The final result should give the parcel address, bus stop name, and distance apart in meters.
+Order by distance (largest on top).
+*/
+
+/* 03-10 using PostgreSQL*/
+
+
+select address,
+       stop_name,
+       distance_m*111000 as distance_m
```

Comment:
Any time you're doing distances with geometry values, you should use a non-lat/lng projection